common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,692 @@
+	}
+	gene_eye_shut = {
+        1 = { name = eye_shut_neg    range = { 0.3 0.5  }    }
+	}
+	gene_bs_eye_corner_depth = {
+        1 = { name = eye_corner_depth_neg    range = { 0 1 }    }
+		1 = { name = eye_corner_depth_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_fold_shape = {
+        1 = { name = eye_fold_shape_pos    range = { 0 0.65 }    }
+	}
+	gene_bs_eye_size = {
+        1 = { name = eye_size_pos    range = { 0 1 }    }
+		1 = { name = eye_size_neg    range = { 0 1 }    }
+	}
+	gene_bs_eye_upper_lid_size = {
+        1 = { name = eye_upper_lid_size_pos    range = { 0 1 }    }
+	}
+	face_detail_eye_lower_lid_def = {
+        1 = { name = eye_lower_lid_def    range = { 0 1 }    }
+	}
+	face_detail_eye_socket = {
+        1 = { name = eye_socket_01    range = { 0 1 }    }
+		1 = { name = eye_socket_02    range = { 0 1 }    }
+		1 = { name = eye_socket_03    range = { 0 1 }    }
+	}
+	
+	###Tusks
+	gene_bs_mouth_tusks = {
+        1 = { name = mouth_tusks    range = { 1 1	  }    }
+    }
+	static_tusks = {
+        1 = { name = troll_tusks    range = { 0 1	  }    }
+    }
+	
+	###Fingers
+	gene_bs_fingers = {
+        1 = { name = fingers_troll    range = { 1 1	  }    }
+    }
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = no_beard 		range = { 0.0 1.0 } }
+	}
+	special_eyebrows = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.2 0.8 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
+	expression_forehead_wrinkles = {
+        1 = {  name = forehead_wrinkles_01             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_02             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_03             range = { 0.1 1 }     }
+	}
+	complexion = {
+        1 = {  name = complexion_1             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_3             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_4             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_5             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_6             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_7             range = { 0.43 0.43 }     }
+	}
```

Comment:
Why it's so fixed? It's actually a good customization option.

Reply:
> 
> 
> Why it's so fixed? It's actually a good customization option.

Fixed in 8bdb0de6d307090a2ef2ba00f1afc05f100fc3fc. Complexions now range from 0 to 1. The reason it was fixed is because the range only affects the color of the portrait's lips, and I didn't feel like it looked good on trolls. But it adds more variety, so it's all good.